test/DomainController.ts

```diff
@@ -72,8 +72,9 @@ describe("DomainController", function () {
   describe("WithdrawalFunds", function () {
```

Comment:
// test
// need to add test for event validation with params 
// need to add balance checks for each user transferring eth
// build you test in correct order 
// first check custom error messages, then check if the function 
// works correctly


// // - 3 points